PWGJE/Tasks/PhiInJetsJE.cxx

```diff
@@ -0,0 +1,412 @@
+    LFhistos.add("hUSS_1D" ,"hUSS_1D", kTH1F, {MinvAxis});
+    LFhistos.add("hUSS_1D_2_3" ,"hUSS_1D_2_3", kTH1F, {MinvAxis});
+
+    LFhistos.add("hLSS" ,"hLSS", kTH3F, {MultAxis,PtAxis,MinvAxis});
+    LFhistos.add("hLSS_1D" ,"hLSS_1D", kTH1F, {MinvAxis});
+    LFhistos.add("hLSS_1D_2_3" ,"hLSS_1D_2_3", kTH1F, {MinvAxis});
+
+    LFhistos.add("hMultFT0M", "hMultFT0M", kTH1F, {MultAxis});
+    
+
+    
+    //EVENT SELECTION
+    eventSelection = JetDerivedDataUtilities::initialiseEventSelection(static_cast<std::string>(c_eventSelections));
+    // trackSelection = JetDerivedDataUtilities::initialiseTrackSelection(static_cast<std::string>(trackSelections));
+
+  
+  } //end of init
+
+  //  double massKa = TDatabasePDG::Instance()->GetParticle(kKPlus)->Mass();  // FIXME: Get from the common header
+
+  double massKa = o2::analysis::pdg::MassKPlus;
+
+
+  using EventCandidates = soa::Join<aod::Collisions, aod::EvSels, aod::Mults, aod::MultZeqs>; // , aod::CentFT0Ms, aod::CentFT0As, aod::CentFT0Cs
+  using TrackCandidates = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA, aod::TrackSelection,
+				    aod::pidTPCFullKa, aod::pidTOFFullKa>;  
+  Filter jetCuts = aod::jet::pt > c_jetPtMin&& aod::jet::r == nround(c_jetR.node() * 100.0f);
+
+
+  //Function for track quality cuts
+  template <typename TrackType>
+  bool trackSelection(const TrackType track){
+    // basic track cuts
+    if (std::abs(track.pt()) < c_trkMinPt)
+      return false;
+    
+    if (std::abs(track.dcaXY()) > c_MaxDCArToPVcut)
+      return false;
+    
+    if (std::abs(track.dcaZ()) > c_MaxDCAzToPVcut)
+      return false;
+    
+    if (c_PrimaryTrack && !track.isPrimaryTrack())
+      return false;
+    
+    if(std::abs(track.tpcNClsFindable()) < c_nFindableTPCClusters)
+      return false;
+    
+    if(std::abs(track.tpcNClsCrossedRows()) < c_nTPCCrossedRows)
+      return false;
+    
+    if(std::abs(track.tpcCrossedRowsOverFindableCls()) > c_nRowsOverFindable)
+      return false;
+    
+    if(std::abs(track.tpcChi2NCl() > c_nTPCChi2))
+      return false;
+    
+    if(std::abs(track.itsChi2NCl() > c_nITSChi2))
+      return false;
+    
+    if (c_ConnectedToPV && !track.isPVContributor())
+      return false;    
+    return true;
+  };
+
+  template <typename T>
+  bool trackPID(const T& candidate)
+  {
+    bool tpcPIDPassed{false}, tofPIDPassed{false};
+    if (std::abs(candidate.tpcNSigmaKa()) < c_nTPCPID) 
+      tpcPIDPassed = true;
+    
+    if (candidate.hasTOF()) {
+      if (std::abs(candidate.tofNSigmaKa()) < c_nTOFPID) {
+        tofPIDPassed = true;
+      }
+    }
+    
+    else 
+      tofPIDPassed = true;
+    
+    if (tpcPIDPassed && tofPIDPassed) {
+      return true;
+    }
+    return false;
+  }
+
+  
+  template <bool IsMC, bool IsMix, typename TracksType, typename JetType>
+  void MinvReconstruction(double mult, const TracksType& trk1, const TracksType& trk2, const JetType& jets) {
```

Comment:
Names of functions and variables should start with small letters.

Reply:
fixed